deploy/crds/toolchain_v1alpha1_nstemplatetier.yaml

```diff
@@ -38,14 +38,104 @@ spec:
                     description: The revision of the corresponding template
                     type: string
                   template:
-                    description: Optional field. Used to specify a custom template
-                    type: string
+                    description: Used to specify a custom template
```

Comment:
just don't forget to update this PR with the latest changes: https://github.com/codeready-toolchain/api/pull/37/commits/40020e5d9feaec34ca65f9726fd7cc6ef836777e

Reply:
ok, let me do that now

Reply:
done in 29c11d5